### DISCUSSION TITLE
Query keeps pagination when using filters

### DISCUSSION THREAD

**Discussion by RazmikSaghoyan** [laravel/nova-issues#3199] [Q&A]
- Laravel Version: 7
- Nova Version: 3
- PHP Version: 7.2

### **Steps to Reproduce:**

- Open a resource
- Go to the second page
- Use a filter where you have less than the per-page limit but you know you should have results
- You have an empty table
- When you use the filter, the query keeps the limit 26 offset 25 from the pagination and returns an empty table even if you have results

**Comment by jbrooksuk** [comment]
@davidhemphill I think we've discussed this issue previously, but I can't remember the decision we made?

**Comment by navidnadali** [comment]
Hi @jbrooksuk @davidhemphill @crynobone is there any plans for this? I think due to our use case we are coming up against it a lot where filtering shows an empty table due to offset not being reset

**Comment by crynobone** [comment]
@navidnadali I marked it as discussion so that we can discussed what's the suitable "Expected Behavior" for this issue, if it's something that the team comfortable with then we could come out with a solution.

**Reply by flatcapco** [reply]
Can this be discussed please? I cant see why the current functionality of it would ever be the expected value as it will always give the user a false positive of no results when there are really results. If this was the intended interaction then there should be clear messaging on the screen reminding them of their position and that there are indeed results.

**Comment by crynobone** [comment]
I just finished adding additional dusk tests and it seems that pagination would reset to page 1 when filters changed or reset. Marking this is a none-issue based on the description above.
